client/lib/checkout/index.js

```diff
@@ -6,23 +6,13 @@ import { parse } from 'url'; // eslint-disable-line no-restricted-imports
 /**
  * Internal dependencies
  */
-import {
-	hasRenewalItem,
-	getRenewalItems,
-	hasDomainRegistration,
-	hasDomainMapping,
```

Comment:
It looks like this is the only place this was used.

Reply:
Good eye, however I think I'm going to leave this around for now as it seems like it might have a utility value later on.